jwt/api_jwt.py

```diff
@@ -87,8 +87,13 @@ def _validate_claims(self, payload, options, audience=None, issuer=None,
         if isinstance(leeway, timedelta):
             leeway = timedelta_total_seconds(leeway)
 
-        if not isinstance(audience, (string_types, type(None))):
-            raise TypeError('audience must be a string or None')
+        if not isinstance(audience, (string_types, type(None), list)):
+            if not isinstance(audience, list):
+                raise TypeError('audience must be a string or None')
```

Comment:
Actually, it looks like you basically already did that for the most part. Maybe we should just get rid of the redundant check here completely?